[2025-02-28T02:20:05.516+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T02:15:00+00:00 [queued]>
[2025-02-28T02:20:05.526+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T02:15:00+00:00 [queued]>
[2025-02-28T02:20:05.526+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-02-28T02:20:05.542+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres_file> on 2025-02-28 02:15:00+00:00
[2025-02-28T02:20:05.550+0000] {standard_task_runner.py:60} INFO - Started process 859 to run task
[2025-02-28T02:20:05.554+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_dag', 'load_to_postgres_file', 'scheduled__2025-02-28T02:15:00+00:00', '--job-id', '142', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmp39edc_6z']
[2025-02-28T02:20:05.559+0000] {standard_task_runner.py:88} INFO - Job 142: Subtask load_to_postgres_file
[2025-02-28T02:20:05.628+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T02:15:00+00:00 [running]> on host 3ca41d2e953f
[2025-02-28T02:20:05.706+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres_file' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T02:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T02:15:00+00:00'
[2025-02-28T02:20:05.708+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-02-28T02:20:05.709+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/scripts/load_to_postgres.py']
[2025-02-28T02:20:05.721+0000] {subprocess.py:86} INFO - Output:
[2025-02-28T02:20:06.846+0000] {subprocess.py:93} INFO - [INFO] Đã tải file '2025/02/28/09-20-03.csv' từ MinIO
[2025-02-28T02:20:06.855+0000] {subprocess.py:93} INFO - [INFO] Đã nạp dữ liệu vào bảng 'raw_coin_data' trong PostgreSQL thành công.
[2025-02-28T02:20:06.856+0000] {subprocess.py:93} INFO - [INFO] Quá trình ETL hoàn tất. Đã xử lý 1 file mới từ MinIO.
[2025-02-28T02:20:06.920+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-02-28T02:20:06.947+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_dag, task_id=load_to_postgres_file, execution_date=20250228T021500, start_date=20250228T022005, end_date=20250228T022006
[2025-02-28T02:20:06.972+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-28T02:20:06.992+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
